cmd/gapir/cc/main.cpp

```diff
@@ -478,6 +478,8 @@ static int replayArchive(core::CrashHandler* crashHandler,
                          gapir::ReplayService* replayArchiveService) {
   std::shared_ptr<MemoryAllocator> allocator = createAllocator();
 
+  const std::string payload = "payload";
```

Comment:
I see you're trying to avoid avoid string literals throughout the code, however this name suggests this variable will actually contain a payload, not just the string "payload".

Reply:
Changed to payloadName